File: atcoder/abc044B_beautiful_strings.nim
# unihernandez22
# https://atcoder.jp/contests/abc044/tasks/abc044_b
# implementation

import strutils, tables

let w = stdin.readLine.strip

var letters = initCountTable[char]()
for c in w:
  letters.inc(c)

for i in letters.values:
  if i mod 2 != 0:
    echo "No"
    quit()
echo "Yes"

File: atcoder/abc044B_beautiful_strings.py
# vicfred
# https://atcoder.jp/contests/abc044/tasks/abc044_b
# implementation
w = input()

english = {}

for ch in w:
    if ch in english:
        english[ch] += 1
    else:
        english[ch] = 1

beautiful = True
for value in english.values():
    if value%2 == 1:
        beautiful = False

if beautiful:
    print("Yes")
else:
    print("No")
